docs/tldr/pages/linux/tmt-run.md

```diff
@@ -0,0 +1,37 @@
+# tmt run
+
+> Execute tmt test steps. By default, all steps are run.
+> See also: `tmt`, `tmt-try`.
+> More information: <https://tmt.readthedocs.io/en/stable/overview.html#run>.
+
+- Run all test steps for each plan:
+
+`tmt run`
+
+- Run selected plans and tests:
+
+`tmt run plan -n {{plan name}} test -n {{test name}}`
+
+- Show what tests a specific plan would run:
+
+`tmt run -vv discover plan -n {{plan name}}`
```

Comment:
```suggestion
`tmt run -v discover plan --name {{plan name}}`
```

For listing test names a single `-v` is sufficient. Let's have this as a second example?